codegen/sharing.adoc

```diff
@@ -0,0 +1,127 @@
+tool will be present in the toolbar, making it easy to re-run this packaging step at any time.
+
+{RAS} will create a jar file in your project's `target` folder:
+
+image::maven-build-target-jar.png[Jar file produced by Maven,role=text-center]
+
+TIP: If you are unable to open your `target` folder, try refreshing it by typing _F5_ when the
+folder is selected.
+
+You can now copy the new jar file and paste a copy anywhere you'd like, either within the {RAS}
+Project Explorer or in your system's file explorer. You can also use drag-and-drop to copy the jar
+file, and there are many applications that can serve as drop targets, including email
+applications, cloud storage solutions like Google Drive and Dropbox, and so on.
+
+== Using Packaged GenTemplates
+
+If you create or obtain a GenTemplate packaged as a Jar file and you want to use it to create
+GenTargets in your model projects, you have two simple options:
+
+1. Copy the jar file into your model project's `lib` folder.
+2. Copy the jar file into a `GenTemplates` folder in a project named `shared` in your {RAS} workspace.
+
+The first option makes the GenTemplates in the jar file available for use with all the models in
+that model project, but not models in other projects. The second option makes those GenTemplates
+available in all your model projects.
+
+Either way, you'll need to create the indicated project and/or folder if they don't already exist.
+
+TIP: If you need to create a `shared` project, select _File -> New -> Project_ in the menu bar, then
+choose _General -> Project_ in the New Project dialog.
+
+With your jar file in either location, {RAS} will find any GenTemplates it contains, and you will
+find those GenTemplates available for use.
+
+TIP: You can change the location of the shared folder in the RepreZen preferences. Select _Window ->
+Preferences_ in the menu bar, and then navigate to _RepreZen -> Code Generation_ and enter your
+desired path. If you start your path with `${workspace_loc}`, as with the factory default, your path
+will be resolved within your {RAS} workspace. But you can leave that off and specify a path anywhere
+in your system.
+
+== Using GenTemplate Projects Pulished to GitHub
+
+Many developers now use GitHub to maintain source code for their projects and, in some cases, to
+make that code available to others. If you publish a {RAS} GenTemplate project on GitHub, others
+have the option of importing your project into their own workspace, thereby making your GenTemplates
+available for use on all models in that workspace.
+
+If you are the publisher of a GenTemplate project in this scenario, you are presumably familiar with
+the steps necessary to set up and publish to GitHub, so we will focus on the steps required to
+import such a project.
+
+You can follow along with the steps below to import the GenTemplate that is developed in link:{{
+"/codegen_intro/" | prepend: site.baseurl }}[Introduction to Code Generation in {RAS}]
```

Comment:
Is there a github repo (this one or another one) that contains the final, complete state of the example GenTemplate developed in the intro?  Could we point the user there, in case she hasn't already completed that intro example?